config/purposes/scrna_core_library_prep_chromium_bcr.yml

```diff
@@ -0,0 +1,83 @@
+# This configuration file is for the Bespoke Chromium Automated pipeline for scRNA core library prep VDJ option BCR
+# library type requests.
```

Comment:
I would take out the 'Bespoke Chromium Automated pipeline' bit from the comment - I know what you're getting at, because this is cloned from that pipeline - but I think we just call this one 'the BCR branch of the scRNA Core library prep pipeline' or something, to avoid confusion.